keylime/tpm/tpm_main.py

```diff
@@ -1039,51 +1039,77 @@ def create_quote(
 
         return "r" + quote
 
-    def __tpm2_checkquote(
-        self, pubaik: str, nonce: str, quoteFile: str, sigFile: str, pcrFile: str, hash_alg: Union[Hash, str]
+    def tpm2_checkquote(
```

Comment:
You changed the function from a private to a public function. You would at least have to adjust all callers as well.